extensions/search-notion/package.json

```
@@ -0,0 +1,58 @@
+{
+  "name": "search-notion",
+  "title": "Search Notion",
+  "description": "Raycast Command for Searching Notion",
```

How about some description like this?

```suggestion
  "description": "Search Notion pages and open visited ones faster. ",
```

---

fixed